test/team_test.rb

```diff
@@ -4,9 +4,9 @@ class TeamTest < Minitest::Test
 
   def setup
     attributes = {team_id: "1",
-                  franchiseId: "23",
```

Comment:
why u do this

Reply:
We needed to adjust the case of the symbols to handle the usage of the header_converter for reading in our CSV files.